tests/test_middleware.py

```diff
@@ -0,0 +1,70 @@
+from unittest.mock import Mock, patch
+
+from django.test import RequestFactory
+
+from vitrina.middleware import AutoRevisionCommentMiddleware
+from vitrina.utils import RevisionComment, RevisionSource
+
+
+def test_noop_when_reversion_not_active(rf: RequestFactory):
+
+    middleware = AutoRevisionCommentMiddleware(get_response=lambda r: None)
+    with patch("vitrina.middleware.reversion.is_active", return_value=False), \
+        patch("vitrina.middleware.reversion.set_comment") as set_comment_mock:
```

Comment:
Use parantheses to avoid slash line-breaks
```suggestion
    with (
        patch("vitrina.middleware.reversion.is_active", return_value=False),
        patch("vitrina.middleware.reversion.set_comment") as set_comment_mock
    ):
```